lib/siwapp_web/templates/page/index.html.heex

```diff
@@ -1,41 +1 @@
-<section class="phx-hero">
-  <h1><%= gettext "Welcome to %{name}!", name: "Phoenix" %></h1>
-  <p>Peace of mind from prototype to production</p>
-</section>
-
-<section class="row">
-  <article class="column">
-    <h2>Resources</h2>
-    <ul>
-      <li>
-        <a href="https://hexdocs.pm/phoenix/overview.html">Guides &amp; Docs</a>
-      </li>
-      <li>
-        <a href="https://github.com/phoenixframework/phoenix">Source</a>
-      </li>
-      <li>
-        <a href="https://github.com/phoenixframework/phoenix/blob/v1.6/CHANGELOG.md">v1.6 Changelog</a>
-      </li>
-    </ul>
-  </article>
-  <article class="column">
-    <h2>Help</h2>
-    <ul>
-      <li>
-        <a href="https://elixirforum.com/c/phoenix-forum">Forum</a>
-      </li>
-      <li>
-        <a href="https://web.libera.chat/#elixir">#elixir on Libera Chat (IRC)</a>
-      </li>
-      <li>
-        <a href="https://twitter.com/elixirphoenix">Twitter @elixirphoenix</a>
-      </li>
-      <li>
-        <a href="https://elixir-slackin.herokuapp.com/">Elixir on Slack</a>
-      </li>
-      <li>
-        <a href="https://discord.gg/elixir">Elixir on Discord</a>
-      </li>
-    </ul>
-  </article>
-</section>
\ No newline at end of file
+<%= Invoices.Graphics.amount_per_day_chart() %>
```

Comment:
The idea would be to call here something like:
`GraphicHelpers.line_plot(Invoices.Graphics.get_data(days: 30))`
I hope to be clear, the idea is not to have cross dependencies between modules

Reply:
Ok, now I understand. I'll try to do it, but the thing is that the data I am producing with those functions is really dependent to the GraphicHelpers, for example I'm changing Date type to NaiveDateTime because of the library limitations, so it's difficult for GraphicHelpers and Invoices.Graphics to be completely independent.

Reply:
I mean that you can reflect that independence separating them in different files, but at the end, they're actually dependent unless I change the code. I could try to do it.

Reply:
My proposal is:
- Invoices.Graphics renamed as Invoices.Statistics, and with function get_data(days) that only elaborates the data but in basic Date type
- GraphicHelpers same as now
- A function called dashboard_chart/0 in the PageController that called Invoices.Statistics.get_data(day), change it to NaiveDateTime type and then call GraphicHelpers.line_plot
- In the index template you leave `<%= dashboard_chart() %>`

Reply:
Ok, but the place to put the fns you are going to use in a template (i.e. dashboard_chart) should go into the views, not in the controllers. Read https://hexdocs.pm/phoenix/views.html

Reply:
Aaaah ok, I had always doubted about that. Thank you.